src/main/java/com/flightstats/hub/app/HubMain.java

```diff
@@ -176,15 +177,18 @@ public Server startServer() throws Exception {
     }
 
     private List<Service> getBeforeHealthCheckServices(Injector injector) {
-
-        final List<Service> services = Stream.of(
+        List<Service> services = createInstanceList(
+                injector,
                 InfluxdbReporterLifecycle.class,
-                StatsDReporterLifecycle.class)
-                .map(injector::getInstance)
-                .collect(Collectors.toList());
+                StatsDReporterLifecycle.class);
 
-        if (storageBackend == StorageBackend.aws && !appProperties.isReadOnly()) {
-            services.add(injector.getInstance(S3WriteQueueLifecycle.class));
+        if (storageBackend == StorageBackend.aws) {
+            if (!appProperties.isReadOnly()) {
+                services.add(injector.getInstance(S3WriteQueueLifecycle.class));
+            }
+            services.addAll(createInstanceList(injector,
+                    DynamoChannelConfigDaoLifecycle.class,
+                    DynamoWebhookDaoLifecycle.class));
         }
```

Comment:
Nice. Eventually this whole service registration thing should be in its own class.

Reply:
These particular ones may even die off.  I don't really think Hub should be responsible for configuring its own AWS resources.